test/test_dgmulti_1d.jl

```diff
@@ -25,7 +25,8 @@ isdir(outdir) && rm(outdir, recursive = true)
         t = sol.t[end]
         u_ode = sol.u[end]
         du_ode = similar(u_ode)
-        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
+        Trixi.rhs!(du_ode, u_ode, semi, t) # run once first to deal with spurious allocations
```

Comment:
Can this line (and the similar ones below) now removed after the latest fixes of the allocations?

Reply:
```suggestion
```
Let's try.

Reply:
Looks like this test passed :tada: